codespan-reporting/assets/readme_preview.svg

```diff
@@ -52,23 +52,23 @@
-<span class="fg blue">15</span> <span class="fg blue">│</span> <span class="fg blue">│</span>         _ 0 =&gt; "Buzz"
-   <span class="fg blue">│</span> <span class="fg blue">│</span>                <span class="fg blue">------</span> <span class="fg blue">this is found to be of type `String`</span>
-<span class="fg blue">16</span> <span class="fg blue">│</span> <span class="fg blue">│</span>         _ _ =&gt; <span class="fg red">num</span>
-   <span class="fg blue">│</span> <span class="fg blue">│</span>                <span class="fg red">^^^</span> <span class="fg red">expected `String`, found `Nat`</span>
-   <span class="fg blue">│</span> <span class="fg blue">╰</span><span class="fg blue">──────────────────' `case` clauses have incompatible types</span>
-   <span class="fg blue">│</span>  
-   <span class="fg blue">=</span> expected type `String`
+   <span class="fg cyan">┌─</span> FizzBuzz.fun:16:16
+   <span class="fg cyan">│</span>  
+<span class="fg cyan">10</span> <span class="fg cyan">│</span>   fizz₂ : Nat → String
+   <span class="fg cyan">│</span>                 <span class="fg cyan">------</span> <span class="fg cyan">expected type `String` found here</span>
+<span class="fg cyan">11</span> <span class="fg cyan">│</span>   fizz₂ num =
+<span class="fg cyan">12</span> <span class="fg cyan">│</span> <span class="fg cyan">╭</span>     case (mod num 5) (mod num 3) of
+<span class="fg cyan">13</span> <span class="fg cyan">│</span> <span class="fg cyan">│</span>         0 0 =&gt; "FizzBuzz"
+   <span class="fg cyan">│</span> <span class="fg cyan">│</span>                <span class="fg cyan">----------</span> <span class="fg cyan">this is found to be of type `String`</span>
+<span class="fg cyan">14</span> <span class="fg cyan">│</span> <span class="fg cyan">│</span>         0 _ =&gt; "Fizz"
+   <span class="fg cyan">│</span> <span class="fg cyan">│</span>                <span class="fg cyan">------</span> <span class="fg cyan">this is found to be of type `String`</span>
+<span class="fg cyan">15</span> <span class="fg cyan">│</span> <span class="fg cyan">│</span>         _ 0 =&gt; "Buzz"
+   <span class="fg cyan">│</span> <span class="fg cyan">│</span>                <span class="fg cyan">------</span> <span class="fg cyan">this is found to be of type `String`</span>
+<span class="fg cyan">16</span> <span class="fg cyan">│</span> <span class="fg cyan">│</span>         _ _ =&gt; <span class="fg red">num</span>
+   <span class="fg cyan">│</span> <span class="fg cyan">│</span>                <span class="fg red">^^^</span> <span class="fg red">expected `String`, found `Nat`</span>
+   <span class="fg cyan">│</span> <span class="fg cyan">╰</span><span class="fg cyan">──────────────────' `case` clauses have incompatible types</span>
+   <span class="fg cyan">│</span>  
+   <span class="fg cyan">=</span> expected type `String`
```

Comment:
Woops, seems like this is being rendered differently on Windows. We need to update the svg example to a platform-consistent configuration (as we do in the integration tests)

Reply:
Should I add that to this PR? Seems quite easy to do.

Reply:
That would be great, if you can! 🤩